dev/reference/configuration/tidb-in-kubernetes/cluster-configuration.md

```diff
@@ -0,0 +1,214 @@
+| `services[0].name` | The name of the service that TiDB cluster exposes | `nil` |
+| `services[0].type` | The type of service that TiDB cluster exposes (selected from `ClusterIP`, `NodePort` and `LoadBalancer`) | `nil` |
+| `discovery.image` | The image of PD's Service Discovery Component in a TiDB cluster. This component is used to provide service discovery for each PD instance to coordinate the starting sequence when the PD cluster is started for the first time. | `pingcap/tidb-operator:v1.0.0-beta.3` |
+| `discovery.imagePullPolicy` | The pulling policy for the image of PD's Service Discovery Component | `IfNotPresent` |
+| `discovery.resoureces.limits.cpu` | The limit of the image of PD's Service Discovery Component on CPU resource | `250m` |
+| `discovery.resoureces.limits.memory` | The limit of the image of PD's Service Discovery Component on memory resource  | `150Mi` |
+| `discovery.resoureces.requests.cpu` | The image of PD's Service Discovery Component's request for CPU resource  | `80m` |
+| `discovery.resoureces.requests.memory` | The image of PD's Service Discovery Component's request for memory resource | `50Mi` |
+| `enableConfigMapRollout` | Whether to enable the automatic rolling update of TiDB cluster. If enabled, TiDB cluster automatically updates the corresponding components when the `ConfigMap` of this cluster changes. This configuration is only supported in `tidb-operator` v1.0 and later versions. | `false` |
+| `pd.config` | The configuration of PD. Check [this link](https://github.com/pingcap/pd/blob/master/conf/config.toml) for the file of the default PD configuration (by choosing the tag of the corresponding PD version). You can see [this document](https://pingcap.com/docs-cn/v3.0/reference/configuration/pd-server/configuration-file/) for the detailed description of the configuration parameters (by choosing the corresponding document version). Here you only need to **modify the configuration based on the format of the configuration file**.  | If the version of TiDB Operator is v1.0.0-beta.3 or earlier, the default value is <br>`nil`<br>If the version of TiDB Operator is later than v1.0.0-beta.3, the default value is <br>`[log]`<br>`level = "info"`<br>`[replication]`<br>`location-labels = ["region", "zone", "rack", "host"]`.<br>Example of configuration:<br>&nbsp;&nbsp;`config:` \|<br>&nbsp;&nbsp;&nbsp;&nbsp;`[log]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`level = "info"`<br>&nbsp;&nbsp;&nbsp;&nbsp;`[replication]`<br>&nbsp;&nbsp;&nbsp;&nbsp;`location-labels = ["region", "zone", "rack", "host"]` |
+| `pd.replicas` | The number of Pods in PD | `3` |
+| `pd.image` | PD image | `pingcap/pd:v3.0.0-rc.1` |
+| `pd.imagePullPolicy` | The pulling policy for PD image | `IfNotPresent` |
+| `pd.logLevel` | The log level of PD<br>If the version of TiDB Operator is later than v1.0.0-beta.3, you can configure the parameter via `pd.config`: <br>`[log]`<br>`level = "info"` | `info` |
```

Comment:
"you can configure" to "you must configure" and all the following should be changed.